skops/card/_model_card.py

```diff
@@ -289,9 +282,39 @@ def save(self, path: str | Path) -> None:
                 model_plot=self._model_plot,
                 **template_sections,
             )
+        return card
+
+    def save(self, path: str | Path) -> None:
+        """Save the model card.
 
+        This method renders the model card in markdown format and then saves it
+        as the specified file.
+
+        Parameters
+        ----------
+        path: str, or Path
+            filepath to save your card.
+
+        Notes
+        -----
+        The keys in model card metadata can be seen `here
+        <https://huggingface.co/docs/hub/models-cards#model-card-metadata>`__.
+        """
+        card = self._generate_card()
         card.save(path)
 
+    def render(self) -> str:
```

Comment:
I liked what you did here but it's actually `_generate_card` that does rendering (with `from_template`) so I'd call this something else like `render_to_str` to avoid confusion for future maintainers. Feel free to ignore if you don't agree 🙂

Reply:
> but it's actually _generate_card that does rendering (with from_template)

I agree that `_generate_card` does the heavy lifting, but from a user perspective, it doesn't really matter.

> so I'd call this something else like render_to_str to avoid confusion for future maintainers

I did think about `render_to_str`. IMO the name should be chosen with respect to the users, not so much maintainers, as it'll be a public method. `render_to_str` makes sense if we plan on adding other outputs and then have `render_to_foo`. If, instead, we want to support those other formats via arguments (`card.render(format="foo")`), I'd stick with the terse `render`. What is the more likely scenario?